chef_master/source/resource_windows_service.rst

```diff
@@ -144,14 +144,14 @@ The windows_service resource has the following properties:
    New in Chef Client 14.0.
 
 ``ignore_failure``
-   **Ruby Types:** True, False
+   **Ruby Types:** true, false | **Default Value:** ``false``
 
-   Continue running a recipe if a resource fails for any reason. Default value: ``false``.
+   Continue running a recipe if a resource fails for any reason.
 
 ``init_command``
-   **Ruby Type:** String
+   **Ruby Type:** String | **Default Value:** ``nil``
```

Comment:
If it's nil we should actually skip the default altogether. Nil is basically no default

Reply:
There are a few of those ``nil``s around. Will make a note of it and fix it.